app/src/main/java/com/powerhungers/fomezero/feature/standbyClass/ConsumerFragment.kt

```diff
@@ -1,24 +1,49 @@
 package com.powerhungers.fomezero.feature.standbyClass
 
+import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import com.powerhungers.fomezero.R
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.powerhungers.fomezero.databinding.FragmentConsumerBinding
 
 class ConsumerFragment : Fragment() {
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-    }
+    private val binding by lazy { FragmentConsumerBinding.inflate(layoutInflater) }
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? {
-        // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_consumer, container, false)
+    ): View {
+        return binding.root
     }
 
-}
\ No newline at end of file
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        handleButtonClick()
+    }
+
+    private fun handleButtonClick() {
+        with(binding){
+            leftOngImageView.setOnClickListener {
+                val intent =
+                    Intent(Intent.ACTION_VIEW).setData(Uri.parse("https://powerhungers.netlify.app/"))
+                startActivity(intent)
+            }
+
+            rightOngImageView.setOnClickListener {
+                val intent =
+                    Intent(Intent.ACTION_VIEW).setData(Uri.parse("https://powerhungers.netlify.app/"))
+                startActivity(intent)
+            }
+
+            makeYourSignature.setOnClickListener {
+                findNavController().navigate(ConsumerFragmentDirections.navigateToSignatureFragment3())
```

Comment:
```suggestion
                findNavController().navigate(ConsumerFragmentDirections.navigateToSignatureFragment())
```
o nome dessa action com numero 3 no final não ficou muito legal